### PR TITLE
Hide "Start Build" actions for binary builds

### PR DESCRIPTION
**Pull request by spadgett**
Uploading a source directory for binary builds is not supported in the web console, so hide the actions that will result in errors.

### CODE REVIEW

File: app/scripts/directives/overview/listRow.js [hunk 1]
```diff
@@ -158,9 +159,14 @@ function OverviewListRow($filter,
         return true;
       }
       // Start pipeline or build is displayed.
-      if ((_.size(row.buildConfigs) === 1 || _.size(row.pipelines) === 1) &&
-           canI('buildconfigs/instantiate')) {
-        return true;
+      if (canI('buildconfigs/instantiate', 'create')) {
```
Comment (jwforres) [inline app/scripts/directives/overview/listRow.js]: we are duplicating a lot of logic from showStartBuildAction and showStartPipelineAction, is there a reason not to just make calls to each of those?

File: app/scripts/filters/resources.js [hunk 2]
```diff
@@ -763,6 +763,12 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .filter('isBinaryBuild', function() {
+    // Accepts a build or build config.
+    return function(build) {
+      return _.get(build, 'spec.source.type') === 'Binary';
```
Comment (jwforres) [inline app/scripts/filters/resources.js]: wasn't spec.source.type deprecated because of the support for multiple sources?

**Comment by spadgett** [comment]
@jwforres updated

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
[merge]

**Pull request merged**